manifests/03_credentials_request_azure.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
     include.release.openshift.io/single-node-developer: "true"
     capability.openshift.io/name: Storage
 spec:
+  serviceAccountNames:
+  - azure-disk-csi-driver-operator
```

Comment:
@jstuever If we need to use the secret with [this](https://github.com/openshift/azure-disk-csi-driver-operator/blob/f71f1449938a1dde07904dc079a3f36284f8e99a/assets/controller.yaml#L26) service account instead, does it need to be added here?

We have to load the secret with [injector](https://github.com/openshift/azure-disk-csi-driver-operator/blob/f71f1449938a1dde07904dc079a3f36284f8e99a/assets/controller.yaml#L60) which will parse it's values and create a cloud.conf for the driver.